backend/handlers/handlers.go

```diff
@@ -1032,6 +1036,44 @@ func (handlers *Handlers) postExportAccountSummary(_ *http.Request) (interface{}
 	return path, nil
 }
 
+func (handlers *Handlers) getExchangesByRegion(r *http.Request) (interface{}, error) {
+	type errorResult struct {
+		Error string `json:"error"`
+	}
+	regions, err := exchanges.ListExchangesByRegion(handlers.backend.HTTPClient())
+	if err != nil {
+		handlers.log.Error(err)
+		return errorResult{Error: err.Error()}, nil
+	}
+	return regions, err
+}
+
+func (handlers *Handlers) getExchangeDeals(r *http.Request) (interface{}, error) {
+	type exchangeDeals struct {
+		Pocket  []exchanges.ExchangeDeal `json:"pocket"`
+		Moonpay []exchanges.ExchangeDeal `json:"moonpay"`
+	}
+	return exchangeDeals{
+		Pocket:  exchanges.PocketDeal(),
+		Moonpay: exchanges.MoonpayDeal(),
+	}, nil
+}
+
+func (handlers *Handlers) getExchangeBuySupported(r *http.Request) (interface{}, error) {
+	acct, err := handlers.backend.GetAccountFromCode(mux.Vars(r)["code"])
+	if err != nil {
+		return nil, err
+	}
+
+	// TODO: Offline() can be removed from here once there is a unified way of initializing accounts
+	// and showing sync status, offline/fatal states, etc.
```

Comment:
please avoid TODOs in the code, more often than not they stay nearly forever. better to create issues in the issue tracker about things that need to be improved than use the codebase as an issue tracker :)